idd/Energy+.idd.in

```diff
@@ -43990,7 +43990,7 @@ ZoneHVAC:LowTemperatureRadiant:VariableFlow,
         \memo Low temperature hydronic radiant heating and/or cooling system embedded in a building
         \memo surface (wall, ceiling, or floor). Controlled by varying the hot or chilled water
         \memo flow to the unit.
-        \min-fields 33
+        \min-fields 34
```

Comment:
OK, so the new input is a required field on the end?

Reply:
Yes